src/main/java/com/twilio/oai/TwilioGoGenerator.java

```diff
@@ -63,6 +63,44 @@ public ModelsMap postProcessModels(final ModelsMap objs) {
         return results;
     }
 
+    @Override
+    public void postProcessModelProperty(CodegenModel model, CodegenProperty property) {
+        super.postProcessModelProperty(model, property);
+        if (property.dataType.startsWith("[]") && property.dataType.contains("Enum")) {
+            property._enum = (List<String>) property.items.allowableValues.get("values");
+            property.allowableValues = property.items.allowableValues;
+            property.dataType = "[]string";
+            property.isEnum = property.dataType == null;
+            property.isNullable = true;
+        } else if (property.dataType.contains("Enum")) {
+            String[] value = property.dataType.split("Enum");
+            property.datatypeWithEnum = value[value.length-1];
+            property.dataType = "string";
+            property.isEnum = property.dataType == null;;
```

Comment:
This would always be false.

Reply:
Yes, This should be property.isEnum = property.dataFormat == null;; Will fix it before merging